packages/@aws-cdk/aws-ecr/lib/repository.ts

```diff
@@ -324,6 +324,13 @@ export interface RepositoryProps {
    *  @default false
    */
   readonly imageScanOnPush?: boolean;
+
+  /**
+   * Enable image tag references to be mutable
+   *
+   *  @default true
```

Comment:
The default for this is actually `false` right? When you pass `undefined` to the L1, then the default value is `MUTABLE` as documented [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-repository.html#cfn-ecr-repository-imagetagmutability)